capstone/src/test/java/com/google/sps/objects/CommentTest.java

```diff
@@ -63,31 +65,11 @@ public void getUserTest() {
   }
 
   @Test
-  public void getCommentEntityTest() {
-    EmbeddedEntity entity = new EmbeddedEntity();
-    entity.setProperty("userId", USER_ID);
-    entity.setProperty("commentText", COMMENT_TEXT);
-    entity.setProperty("timestamp", TIMESTAMP);
-    Comment returnedComment = Comment.getCommentEntity(entity);
+  public void toAndFromCommentEntityTest() {
```

Comment:
I know Himani was having some issues with this test because some data needed to be stored in the DB to be formatted correctly, so double check this is working as intended and passing. If it isn't, you likely just need to follow what she did for the toFromEntityTest in Objects/UserTest.java

Reply:
I checked again and my tests are working as intended. The correct values are being stored in toEntity and fromEntity. I'll ask her what wasn't passing just in case.